arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/gas/projector/Create.java

```diff
@@ -19,14 +19,17 @@
 
 import net.consensys.linea.zktracer.Trace;
 import org.hyperledger.besu.evm.frame.MessageFrame;
+import org.hyperledger.besu.evm.gascalculator.GasCalculator;
 import org.hyperledger.besu.evm.internal.Words;
 
 public final class Create extends GasProjection {
+  final GasCalculator gc;
```

Comment:
What's the point of this ? AFAICT it's not used here (nor in the other methods.) Does it give us access to the hard fork ?